cmd/describe-compiler.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/emc-advanced-dev/pkg/errors"
+	"github.com/emc-advanced-dev/unik/pkg/client"
+)
+
+var describeCompilerCmd = &cobra.Command{
+	Use:   "describe-compiler",
+	Short: "Describe compiler usage",
+	Long: `Describes compiler usage.
+You must provide triple base-language-provider to specify what compiler to describe.`,
```

Comment:
I am still confused whether the `provider` is actually required here. Does any of the manifests rely on this info?